_posts/RhymesWithMecca/2016-04-28-BeccasFinalProject.md

```diff
@@ -0,0 +1,31 @@
+
+In my mid-semester reflection, I talked about being nervous to experiment because I was worried I’d screw up and hoping I could get better at this as time goes on.  I do think I’ve gotten better at this since writing that (yay!).  I’m more comfortable with having a very rough program, with a lot of test print statements and commented-out incorrect code, since getting things wrong is crucial to figuring out how to get them right.  I also think that ways to troubleshoot, potential solutions, or alternative ways to code things occur to me faster than they used to.  
+
+One thing that made this project hard was the lack of color variety in modules that aren’t “main.py.”  To troubleshoot the user checking above, I moved the cDNA translate function from “functions.py” into “main.py” so the colors would show up and I could catch my mistakes faster.  Have Python 3 Trinkets always been like this?  Another was the difference in error encoding in Python 2 vs Python 3 Trinkets – Python 2 highlights your error on the appropriate line and is a lot clearer.  Python 3 traces the problem back through each function before arriving at the error at the end of the statement, which is less intelligible to me (but it might just be me).
```

Comment:
This is a bug that another student say too, but I'm not able to reproduce.  Could you document the steps you take that had this happen?  A workaround is if you edit the .py filenames and then click away the syntax highlighting will appear.  But I'm not sure why they don't for some users.